data/korath/korath weapons.txt

```diff
@@ -973,7 +973,7 @@ outfit "Firelight Turning"
 		"trigger radius" 30
 		"blast radius" 60
 		"hit force" 60
-		"missile strength" 60
+		"missile strength" 30
```

Comment:
Maybe something more like 40 would be better.

Reply:
These missiles are also very fast, so they've got the added defense that AM doesn't get much time to shoot at them. So, I don't think dropping it to 30 would make them excessively vulnerable to AM.

Reply:
You made a good point. 30 is good for me then.